rspace/src/main/scala/coop/rchain/rspace/util/package.scala

```diff
@@ -19,4 +21,38 @@ package object util {
     */
   def runKs[T](t: Seq[Option[((T) => Unit, T)]]): Unit =
     t.foreach { case Some((k, data)) => k(data); case None => () }
+
+  /**
+    * Compare to `memcmp` in C/C++
+    *
+    * Based on:
+    * [[https://github.com/OpenTSDB/asynchbase/blob/f6a8ccb7e55ed9bc0aad265345da4c679e750055/src/Bytes.java#L549-L572]]
+    */
+  @SuppressWarnings(Array("org.wartremover.warts.Equals", "org.wartremover.warts.Return"))
+  def memcmp(a: Array[Byte], b: Array[Byte]): Int = {
```

Comment:
Why not `compare`?

Reply:
Oh it's specifically aping `memcmp` semantics.